opba-banking-rest-api-ymls/src/main/resources/static/tpp_banking_api_bank_search.yml

```diff
@@ -81,7 +83,18 @@ paths:
 components:
   headers: {}
 
-  parameters: {}
+  parameters:
+
+    onlyActive:
+      name: onlyActive
+      in: query
+      required: false
+      description: |
+        Makes possible to get not active item.
```

Comment:
`Makes possible to get not active item.` -> `Controls visibility of inactive banks in the response (false - both active and inactive, true - only active)`

Reply:
fixed